apps/odyssey-stats/webpack.config.js

```diff
@@ -41,9 +41,10 @@ const excludedPackages = [
 ];
 
 const excludedPackagePlugins = excludedPackages.map(
-	( package ) =>
+	// Note: apparently the word "package" is a reserved keyword here for some reason
+	( pkg ) =>
 		new webpack.NormalModuleReplacementPlugin(
-			package,
+			pkg,
```

Comment:
Weird issue, but can consistently replicate when running the translation script in **happy blocks** of all places

Reply:
Nice catch!